### PR TITLE
update container name

### PR DESCRIPTION
**Pull request by wfurt**
This will hopefully fix 

```
2019-02-27 18:03:58,750: INFO: executor(118): run: Successfully downloaded work item payload
2019-02-27 18:03:58,751: INFO: dockerhelper(44): pull: Pulling image dotnet-buildtools-prereqs:alpine-3.9-helix-34f2903-20190227035445 from registry microsoft...
2019-02-27 18:03:59,637: ERROR: executor(521): _execute_command_in_container: Exception: 404 Client Error: Not Found ("manifest for microsoft/dotnet-buildtools-prereqs:alpine-3.9-helix-34f2903-20190227035445 not found")
```

### CODE REVIEW

File: eng/pipelines/linux.yml [hunk 1]
```diff
@@ -89,7 +89,7 @@ jobs:
         - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
           - linuxDefaultQueues: Centos.7.Amd64+RedHat.7.Amd64+Debian.8.Amd64+Debian.9.Amd64+Ubuntu.1604.Amd64+Ubuntu.1804.Amd64+Ubuntu.1810.Amd64+OpenSuse.42.Amd64+SLES.12.Amd64+SLES.15.Amd64+Fedora.27.Amd64+Fedora.28.Amd64
           - linuxArm64Queues: Ubuntu.1604.Arm64
-          - alpineQueues: Alpine.36.Amd64+Alpine.38.Amd64+ubuntu.1604.amd64@microsoft/dotnet-buildtools-prereqs:alpine-3.9-helix-34f2903-20190227035445
+          - alpineQueues: Alpine.36.Amd64+Alpine.38.Amd64+ubuntu.1604.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.9-helix-34f2903-20190227035445
```
Comment (safern) [inline eng/pipelines/linux.yml]: Why is this required? The docker tag is microsoft/dotnet-buildtools-....

Comment (MattGal) [reply eng/pipelines/linux.yml]: When an image is not hosted on dockerhub, it needs to be qualified with its registry, in this case MCR.

Comment (safern) [reply eng/pipelines/linux.yml]: Oh I see. So now we publish the new images from prereqs-docker repo to MCR?

Comment (MichaelSimons) [reply eng/pipelines/linux.yml]: correct - look for an announcement on this soon - I wanted to make sure everything was working correctly before broadcasting this widely.
